gdcdictionary/python/__init__.py

```diff
@@ -73,6 +73,10 @@ def load_yaml(self, name):
                 except Exception as e:
                     self.logger.error("Error in file: {}".format(name))
                     raise e
+            if yaml.__with_libyaml__:
+                return yaml.load(f, Loader=yaml.CSafeLoader)
+            self.logger.warning(e)
+            self.logger.warning("To enable CSafeLoader install libyaml. Falling back to yaml.safe_load()")
```

Comment:
A warning means that something went wrong.  Nothing is wrong here.  `debug` would be an appropriate log level.